s4sim/hardware/config.py

```diff
@@ -1747,14 +1747,14 @@ def sim_nominal():
 
     #from the DSR
```

Comment:
The old and the new values cannot both be from DSR, can they?

Reply:
That is a very stale comment--I just updated and deleted it